src/pages/gd/Claim/ClaimBalance.tsx

```diff
@@ -35,13 +33,14 @@ const ClaimTimer = () => {
 export const ClaimBalance = () => {
     const { claimTime } = useClaim('everyBlock')
     const { chainId } = useActiveWeb3React()
-    // const [G$Price] = usePromise(
-    //     () =>
-    //         g$Price()
-    //             .then(({ DAI }) => DAI)
-    //             .catch(() => undefined),
-    //     [chainId]
-    // )
+    const [G$Price] = usePromise(
+        () =>
+            g$Price()
+                .then(({ DAI }) => +DAI.toSignificant(6))
```

Comment:
what type is DAI? (ie original G$Price)
why aren't you the original G$Price to balancegd and let balancegd handle the display?

Reply:
Dai is fraction so here I convert it .toSignifcant returns bigNumber
the + in front makes it a number

Reply:
@sirpy is this a suggestion or a 'should be done' comment?
I just think that passing down number is easier, and causes least amount of inconsistencies and/or issues

Reply:
It's a suggestion. The last rendering component should be in charge of the display